plugins/inputs/win_eventlog/util_test.go

```diff
@@ -0,0 +1,192 @@
+//+build windows
+
+//revive:disable-next-line:var-naming
+// Package win_eventlog Input plugin to collect Windows Event Log messages
+package win_eventlog
+
+import (
+	"bytes"
+	"encoding/binary"
+	"encoding/xml"
+	"io"
+	"reflect"
+	"testing"
+	"unicode/utf16"
+)
+
+func TestDecodeUTF16(t *testing.T) {
+	testString := "Test String"
+	utf16 := utf16.Encode([]rune(testString))
```

Comment:
I know it is just a test but `utf16` variable collides with imported package name.